package.json

```diff
@@ -15,6 +15,9 @@
   "homepage": "https://github.com/ManageIQ/manageiq#readme",
   "dependencies": {
     "@manageiq/react-ui-components": "~0.9.5",
+    "chai": "^4.1.2",
```

Comment:
Is there any reason why you want to use this library? I don't think you will need this at all. Jest already has pretty good assertions.

Reply:
Cool, was looking at enzyme example and they are using it. Let me remove.